codeclimate-govet.go

```diff
@@ -35,12 +50,14 @@ func main() {
 				pieces := strings.Split(line, ":")
 
 				if len(pieces) < 3 {
-					// engine.PrintWarning()
+					fmt.Fprintf(os.Stderr, "Error parsing the following output: %v\n", out)
+					fmt.Fprintf(os.Stderr, "\nError: %v\n", err)
 					return
 				} else {
 					lineNo, err := strconv.Atoi(pieces[1])
 					if err != nil {
-						// engine.PrintWarning()
+						fmt.Fprintf(os.Stderr, "Error parsing the following output: %v\n", out)
+						fmt.Fprintf(os.Stderr, "\nError: %v\n", err)
```

Comment:
Similarly here I think there's only one error we expect, which is that the string couldn't be coerced into an integer, so we could probably just say something about an unexpected format